python/oneflow/framework/docstr/bmm.py

```diff
@@ -28,26 +28,17 @@ def bmm_op(input, mat2):
     Args:
         input(oneflow.Tensor):  the first batch of matrices to be multiplied
         mat2(oneflow.Tensor): the second batch of matrices to be multiplied
-    
+
     For example:
 
     .. code-block:: python
 
         >>> import oneflow as flow
         >>> import numpy as np
-        >>> input1 = flow.Tensor(np.random.randn(10, 3, 4))
-        >>> input2 = flow.Tensor(np.random.randn(10, 4, 5))
+        >>> input1 = flow.Tensor(flow.randn(10, 3, 4))
```

Comment:
不是这个意思，是改成就可以了

```suggestion
        >>> input1 =flow.randn(10, 3, 4)
```

有空好好去读一下官方文档，知悉一下Tensor和tensor的区别。

Reply:
好！我看错了